src/patternfly/components/Content/content.scss

```
@@ -80,12 +80,12 @@
   --#{$content}--blockquote--BorderInlineStartWidth: var(--pf-t--global--border--width--300); // TODO semantic value?
 
   // Lists
-  --#{$content}--list--Gap: var(--pf-t--global--spacer--sm);
+  --#{$content}--list--Gap: var(--pf-t--global--spacer--gap--group--vertical);
   --#{$content}--list--PaddingInlineStart: var(--pf-t--global--spacer--lg);
   --#{$content}--list--nested--MarginBlockStart: var(--#{$content}--list--Gap);
   --#{$content}--ul--ListStyle: var(--pf-t--global--list-style);
-  --#{$content}--dl--ColumnGap: var(--pf-t--global--spacer--2xl);
-  --#{$content}--dl--RowGap: var(--pf-t--global--spacer--md);
+  --#{$content}--dl--ColumnGap: var(--pf-t--global--spacer--gap--group-to-group--horizontal);
+  --#{$content}--dl--RowGap: var(--pf-t--global--spacer--gap--group-to-group--vertical); // Increases the current gap
```

Met with @lboehling today and we looked at the content `<dl>` and description-list component (for another PR - https://github.com/patternfly/patternfly/pull/6953/files) and the row-gap here is good but the column gap should be  `--pf-t--global--spacer--gap--group--horizontal`

```suggestion
  --#{$content}--dl--ColumnGap: var(--pf-t--global--spacer--gap--group--horizontal);
  --#{$content}--dl--RowGap: var(--pf-t--global--spacer--gap--group-to-group--vertical);
```